SpatialGDK/Source/SpatialGDK/Public/Schema/Interest.h

```diff
@@ -117,8 +122,9 @@ struct Query
 	QueryConstraint Constraint;
 
 	// Either full_snapshot_result or a list of result_component_id should be provided. Providing both is invalid.
-	TSchemaOption<bool> FullSnapshotResult; // Whether all components should be included or none.
-	SchemaResultType ResultComponentIds;	// Which components should be included.
+	TSchemaOption<bool> FullSnapshotResult;			  // Whether all components should be included or none.
```

Comment:
indentation is strange- also what does it mean for this optional to be unset?

Reply:
indentation just looks strange on github - has been formatted
Have updated the comment above with the comments from standard_library.schema